icu4c/source/data/misc/supplementalData.txt

```diff
@@ -35597,398 +35597,297 @@ supplementalData:table(nofallback){
-            length-person{"inch"}
-            length-person-informal{"foot inch"}
-            length-person-small{"inch"}
-            length-rainfall{"inch"}
-            length-road{"mile"}
-            length-road-small{"foot"}
-            length-snowfall{"inch"}
-            length-vehicle{"foot inch"}
-            length-visiblty{"mile"}
-            length-visiblty-small{"foot"}
-            mass-person{"pound"}
-            mass-person-small{"pound ounce"}
-            pressure-baromtrc{"inch-hg"}
-            speed-road-travel{"mile-per-hour"}
-            speed-wind{"mile-per-hour"}
-            temperature-person{"fahrenheit"}
-            temperature-weather{"fahrenheit"}
-            volume-vehicle-fuel{"gallon"}
+        "duration"{
+            "music-track"{
+                001{
+                    minute+second{""}
+                }
+            }
+            "person-age"{
+                001{
+                    year-person+month-person{""}
+                }
+            }
+            "tv-program"{
+                001{
+                    minute+second{""}
+                }
+            }
+            _default{
+                001{
+                    year{
+                        geq:1{""}
+                    }
+                    month{
+                        geq:1{""}
+                    }
+                    day{
+                        geq:1{""}
+                    }
+                    hour{
+                        geq:1{""}
+                    }
+                    minute{
+                        geq:1{""}
+                    }
+                    second{
+                        geq:1{""}
+                    }
+                    millisecond{
+                        geq:1{""}
+                    }
+                    microsecond{
+                        geq:1{""}
+                    }
+                    nanosecond{""}
+                }
+            }
         }
-        VC{
-            concentr-blood-glucose{"millimole-per-liter"}
+	// DEMO: alt{"informal"}:
+        "length"{
+            "road"{
+                001{
+                    kilometer{""}
+                }
+                GB{
+                    mile{""}
+                }
+                SE{
+                    mile-scandinavian{
+                        alt{"informal"}
+                    }
+                }
+                US{
+                    mile{""}
+                }
+		// DEMO: Current hack for _small, proposing we drop this info and simply use geq:
+                _small{
+                    001{
+                        meter{""}
+                    }
+                    GB{
+                        yard{""}
+                    }
+                    US{
+                        foot{""}
+                    }
+                }
+            }
+            "road-travel"{
+                001{
+                    kilometer{
+                        geq:0.975{""}
```

Comment:
The 0.975 should be the string value, like this:

```
geq{"0.975"}
```

Reply:
Ah... currently not a string yet. There's an :int type but I don't find something like :float or :double - so floats have to be passed as strings?